<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIcoachDemoClass</class>
 <widget class="QMainWindow" name="AIcoachDemoClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AIcoacher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_main">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_viewers">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_original">
          <item>
           <widget class="QLabel" name="label_original">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>Video Stream</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="video_original">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>500</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_processed">
          <item>
           <widget class="QLabel" name="label_processed">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>Skeleton Points</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="video_processed">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>500</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_count">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_count_num">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_count_max">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>/ 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string>Time: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time_num">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_time_max">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>/ 0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_prompt_content">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_right">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>90</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="text">
              <string>Curriculum: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_class">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Side Lateral Raise</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y-stretch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Squat</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_start">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>START</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_suspend">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>PAUSE/RESUME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_stop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>STOP</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_log">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_log">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="action_load_file"/>
    <addaction name="action_load_camera"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_save_video">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存检测视频</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_save_data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存动作序列</string>
   </property>
  </action>
  <action name="action_load_file">
   <property name="text">
    <string>Load video file</string>
   </property>
  </action>
  <action name="action_load_camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load webcam</string>
   </property>
  </action>
  <action name="action_etract">
   <property name="text">
    <string>_etract</string>
   </property>
  </action>
  <action name="action_extract">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>动作提取</string>
   </property>
  </action>
  <action name="action_train">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>动作训练</string>
   </property>
  </action>
  <action name="action_class_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>课程编辑</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AIcoachDemo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>menuExitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_start</sender>
   <signal>clicked()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>pushButtonStartClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_stop</sender>
   <signal>clicked()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>pushButtonEndClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_suspend</sender>
   <signal>clicked()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>pushButtonSuspendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_load_file</sender>
   <signal>triggered()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>menuFileLoadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_load_camera</sender>
   <signal>triggered()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>menuCameraLoadClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_about</sender>
   <signal>triggered()</signal>
   <receiver>AIcoachDemoClass</receiver>
   <slot>menuAboutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>menuExitClicked()</slot>
  <slot>pushButtonStartClicked()</slot>
  <slot>pushButtonEndClicked()</slot>
  <slot>pushButtonSuspendClicked()</slot>
  <slot>menuFileLoadClicked()</slot>
  <slot>menuCameraLoadClicked()</slot>
  <slot>menuAboutClicked()</slot>
 </slots>
</ui>
